python/requirements.txt

```diff
@@ -0,0 +1,59 @@
+# General requirements
+blist; platform_system != "Windows"
+boto3
+cython==0.29.0
+dataclasses
+feather-format
+gym
+kubernetes
+lxml
+mypy
+networkx
+numba
+openpyxl
+pettingzoo
+Pillow; platform_system != "Windows"
+pygments
+pytest
+pytest-asyncio
+pytest-rerunfailures
+pytest-sugar
+pytest-timeout
+scikit-learn==0.22.2
+tensorflow==2.2.0
+werkzeug
+xlrd
+
+# setup.py requirements
```

Comment:
Does this need to be kept in sync with setup.py? If so it'd be nice to note that explicitly in both places (and similarly elsewhere).

Reply:
This is the only place that really needs syncing